monkey/monkey_island/cc/resources/root.py

```diff
@@ -7,6 +7,7 @@
 from monkey_island.cc.auth import jwt_required
 from monkey_island.cc.database import mongo
 from monkey_island.cc.services.config import ConfigService
+from monkey_island.cc.services.attack.attack_config import reset_config as reset_attack_config
```

Comment:
why? If I only need one method from the whole package...

Reply:
from-import-as clauses tend to be more confusing. there is no function 'reset_attack_config'. The reader has to go to this part of the file to understand what function we're talking about.
using a from-import here is even more confusing since there's another common function with the exact same name.
There's no 'waste' in importing a module vs importing a single function. The whole module is being loaded anyways. The most meaningful difference in this context is how we access the functions etc of the other module.